xorg_xmessage.rb

```diff
@@ -0,0 +1,18 @@
+require 'package'
+
+class Xorg_xmessage < Package
+  description 'Xmessage displays a message or query in a window.'
+  homepage 'https://gitlab.freedesktop.org/xorg/app/xmessage'
+  version 'e315069848aea3aaeb72bd3bc9a5d0bdb1242ed7'
+  source_url 'https://gitlab.freedesktop.org/xorg/app/xmessage/-/archive/e315069848aea3aaeb72bd3bc9a5d0bdb1242ed7/xmessage-e315069848aea3aaeb72bd3bc9a5d0bdb1242ed7.tar.gz'
+  source_sha256 'f2ce4750a60cf8e4a3c75ad542b94e5b3ae9754ef161534c9921c7ef5cd4843c'
+
+  def self.build
+    system "./autogen.sh"
+    system "make PREFIX=#{CREW_PREFIX} LIBDIR=#{CREW_LIB_PREFIX}"
```

Comment:
Remove ` PREFIX=#{CREW_PREFIX} LIBDIR=#{CREW_LIB_PREFIX}` and change to single quotes.